core/templates/dev/head/pages/exploration_editor/editor_tab/StateResponses.js

```diff
@@ -564,16 +566,18 @@ oppia.controller('StateResponses', [
           'explorationContextService', 'editorContextService',
           'explorationStatesService', 'trainingDataService',
           'AnswerClassificationService', 'focusService',
-          'RULE_TYPE_CLASSIFIER',
+          'RULE_TYPE_CLASSIFIER', 'oppiaPlayerService',
```

Comment:
Oh, ok. I was confused between the two. But then what is the rationale behind deciding which service can be used in another service? (as in the case with oppiaPlayerService)